src/Agent/NewRelic/Agent/Core/DistributedTracing/DistributedTracePayloadHandler.cs

```diff
@@ -162,12 +162,15 @@ private string BuildTracestate(IInternalTransaction transaction, DateTime timest
             var newRelicTracestate = $"{accountKey}@nr={version}-{parentType}-{parentAccountId}-{appId}-{spanId}-{transactionId}-{sampled}-{priority}-{timestampInMillis}";
             var otherVendorTracestates = string.Empty;
 
-            if (transaction.TracingState != null)
+            if (transaction.TracingState?.VendorStateEntries != null)
```

Comment:
You might also do a `&& transaction.TracingState?.VendorStateEntries.Any()` and refactor the `if` block on line 171.

Reply:
I'll take a look!

Reply:
I updated the if on line 165 with the additional check.
I left the if on line 171 as it was since an empty item in VendorStateEntries could still trigger a trailing comma.  That should never happen, but better to be safe and check.